app/main.py

```diff
@@ -2,0 +1,25 @@
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError(
+            "Invalid command format. Use: mv <source> <destination>"
+        )
+
+    source, destination = parts[1], parts[2]
+
+    if not os.path.isfile(source):
+        raise FileNotFoundError(f"Source file '{source}' does not exist.")
+
+    if destination.endswith("/"):
```

Comment:
The condition `destination.endswith("/")` is used to determine if the destination is a directory. This is a good approach, but it might be more robust to use `os.path.isdir(destination)` if the destination path already exists.